src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -271,18 +271,18 @@ This connection registry implementation is responsible for fetching the followin
 * Cluster ID (unique to this cluster)
 
 This information is needed as a part of various client operations like connection set up, scans,
-gets etc. Up until releases 2.x.y, the default connection registry is based on ZooKeeper as the
-source of truth and the the clients fetched the metadata from zookeeper znodes. As of release 3.0.0,
-the default implementation for connection registry has been switched  to a master based
-implementation. With this change, the clients now fetch the required metadata from master RPC end
-points directly. This change was done for the following reasons.
+gets, etc. Traditionally, the connection registry implementation has been based on ZooKeeper as the
+source of truth and clients fetched the metadata directly from the ZooKeeper quorum. HBase 2.3.0
```

Comment:
I couldn't find the ticket -- if this was backported to branch-1, this line could be updated to say "HBase 1.7.0 and 2.3.0 introduce a new connection registry implementation..."

Reply:
I didn't backport the patches yet. Will do it in the coming weeks. Will revisit this section once the backport is done.